src/components/ReportActionAvatars/useReportActionAvatars.ts

```diff
@@ -104,6 +105,7 @@ function useReportActionAvatars({
         type: CONST.ICON_TYPE_AVATAR,
         source: personalDetails?.[id]?.avatar ?? FallbackAvatar,
         name: personalDetails?.[id]?.[shouldUseActorAccountID ? 'displayName' : 'login'] ?? '',
+        fallbackIcon: RandomAvatarUtils.getAvatarForContact(String(id)),
```

Comment:
I think we need to use UserUtils function `getDefaultAvatarURL` here as it relies on the account id.

https://github.com/Expensify/App/blob/e05a55f66216174a96583d9ddbc5bd4425609787/src/libs/UserUtils.ts#L149

With this change, everywhere it renders fallbackIcon and won't use FallbackAvatar. I confirmed that we only need this for Workspace members invite screen. For that, let's add a prop something called `shouldUseFallbackDefaultAvatar` and pass it from the `WorkspaceInviteMessagePage`

Reply:
OK, let it be `getAvatarForContact`.

> With this change, everywhere it renders fallbackIcon and won't use FallbackAvatar. I confirmed that we only need  this for Workspace members invite screen. For that, let's add a prop something called shouldUseFallbackDefaultAvatar and pass it from the WorkspaceInviteMessagePage

We still need this change.